.azuredevops/platformPipelines/platform.updateReadMe.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           pwsh: true
           script: |
             # Load used functions
-            . (Join-Path '$(System.DefaultWorkingDirectory)' 'utilities' 'tools' 'Set-ReadMeModuleTable.ps1')
+            . (Join-Path '$(System.DefaultWorkingDirectory)' 'utilities' 'tools' 'platform' 'Set-ReadMeModuleTable.ps1')
```

Comment:
Should the pipelines be triggered on changes to scripts in the 'utilities/tools/platform' folder?

Reply:
Sure we can add a trigger. Note however, this will also affect a script that is out of scope of this PR - the `Set-ReadMeModuleTable` script

Reply:
Regarding the environment variable I'm still not sold on the idea :D But sure I could set it somewhere. The only problem with the path in the environment would be that I have to choose whether to use `/` or `\` in the path. 'Join-Path' does that for me, based on the used OS.
Thoughts?

Reply:
For the time being I added them, though I'm still not sure. Also, how far do we want to push this? For example, we also have the snippet:

```PowerShell
$functionInput = @{
   MarkdownFilePath = Join-Path '$(System.DefaultWorkingDirectory)' 'docs' 'wiki' 'The library - Module overview.md'
   ModuleFolderPath = Join-Path '$(System.DefaultWorkingDirectory)' 'modules'
}
```

Should these then also be `env` values? Depending on the file the list 'may' get rather long.

Reply:
I agree, let's discuss a consistent way forward offline as it impacts the whole repo. I just would like to avoid hardcoded values as much as possible, but I agree we should probably end up with a reasonable compromise. For this PR I'm fine either way.